CMakeLists.txt

```diff
@@ -441,9 +441,13 @@ include_directories(
 	${LIBXML2_INCLUDE_DIR}
 )
 
+# Honor visibility properties for all target types
+# Run "cmake --help-policy CMP0063" for policy details
+cmake_policy(SET CMP0063 NEW)
+
 function(set_oscap_generic_properties TARGET_OBJECT)
 	set_target_properties(${TARGET_OBJECT} PROPERTIES
-		CXX_VISIBILITY_PRESET hidden    # Make global variables and functions HIDDEN by default.
+		C_VISIBILITY_PRESET hidden    # Make global variables and functions HIDDEN by default.
```

Comment:
Might as well add in those two spaces...

Reply:
we want both IMO because dpkg probe is in C++

Reply:
OK